tasks/daemons.js

```diff
@@ -70,11 +70,13 @@ function startDisposableDaemons (callback) {
               throw err
             }
 
-            ipfsNodes[key].startDaemon((err, ignore) => {
+            ipfsNodes[key].startDaemon((err, daemon) => {
               if (err) {
                 throw err
               }
-
+              daemon.add('test/test-folder', {r: true}, (e, r) => {
```

Comment:
This folder is added too during the `add` tests, could you use the hash from there?